gammapy/estimators/tests/test_flux_point_estimator.py

```diff
@@ -433,29 +433,29 @@ def test_run_pwl_parameter_range(fpe_pwl):
     fp = fpe.run(datasets)
     table_no_bounds = fp.to_table()
 
-    pl.amplitude.min=0
-    pl.amplitude.max=1e-8
+    pl.amplitude.min = 0
+    pl.amplitude.max = 1e-12
 
     fp = fpe.run(datasets)
     table_with_bounds = fp.to_table()
 
     actual = table_with_bounds["norm"].data
-    assert_allclose(actual, [3.215947e-02, 3.939055e-02, 5.551115e-09], rtol=1e-3)
+    assert_allclose(actual, [0., 0., 0.], atol=1e-2)
 
-    actual = table_with_bounds["norm_err"].data
-    assert_allclose(actual, [251.490704, 280.37361 , 404.162784], rtol=1e-2)
+    actual = table_with_bounds["norm_errp"].data
+    assert_allclose(actual, [212.593368, 298.383045, 449.951747], rtol=1e-2)
```

Comment:
As the minimizer is not in a minimum the covariance is not a stable estimate. I change to the confidence based error instead.